tools/generator/Java.Interop.Tools.Generator.ObjectModel/ClassGen.cs

```diff
@@ -269,7 +261,7 @@ public static void GenerateTypeRegistrations (CodeGenerationOptions opt, Generat
 			using (var sw = gen_info.OpenStream (opt.GetFileName ("Java.Interop.__TypeRegistrations"))) {
 
 				Dictionary<string, List<KeyValuePair<string, string>>> mapping = new Dictionary<string, List<KeyValuePair<string, string>>> ();
-				foreach (KeyValuePair<string, string> reg in gen_info.TypeRegistrations) {
+				foreach (KeyValuePair<string, string> reg in gen_info.TypeRegistrations.OrderBy (p => p.Key, StringComparer.OrdinalIgnoreCase)) {
```

Comment:
Note that this adds an ordering to the types in `Java.Interop.__TypeRegistrations.cs`.  This is because the order is based on the order in which we wrote the type `.cs` files, which is no longer deterministic because we are running them in parallel.  I think this is fine and does not impact runtime, but wanted to call it out just in case.  I ran a HelloWorld app and it still worked.

Reply:
I likewise don't think this should matter, *particularly* since this should be dead code *anyway* now that we have "type map" files (since 2015!); see also https://github.com/xamarin/monodroid/commit/e69b76e2b86f92fa8440756c44ad3e2205563ac7